rcl/include/rcl/context.h

```diff
@@ -240,6 +240,31 @@ RCL_WARN_UNUSED
 rcl_context_instance_id_t
 rcl_context_get_instance_id(rcl_context_t * context);
 
+/// Returns the context domain id.
+/**
+ * The given context must be non-`NULL` and be valid.
+ * If context is `NULL`, then `0` will be returned.
```

Comment:
@ivanpauno why this behavior? I'd expect a no-op and `RCL_RET_INVALID_ARGUMENT` as return code.

Reply:
I copied the docblock of rcl_context_get_instance_id and I then forgot to delete that line.